src/frontend/context/ManglendeBrevVariablerContext.ts

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+
+import constate from 'constate';
+
+import { MalStruktur, Valg, Valgfelt, Variabel } from '../komponenter/Brev/typer';
+import { harIkkeVerdi } from '../utils/utils';
+
+export const [ManglendeBrevVariablerProvider, useManglendeBrevVariabler] = constate(() => {
+    const [manglendeBrevVariabler, settManglendeBrevVariabler] = useState<Variabel[]>([]);
+
+    const finnManglendeBrevVariabler = (
+        mal: MalStruktur,
+        inkluderteDelmaler: Record<string, boolean>,
+        valgfelt: Partial<Record<string, Record<Valgfelt['_id'], Valg>>>,
+        variabler: Partial<Record<string, string>>
+    ): Variabel[] => {
+        return mal.delmaler
+            .filter((delmal) => inkluderteDelmaler[delmal._id])
+            .flatMap((delmal) => Object.values(valgfelt[delmal._id] ?? {}))
+            .filter((valg) => valg._type === 'tekst')
+            .flatMap((valg) => valg.variabler)
+            .filter((variabel) => harIkkeVerdi(variabler[variabel._id]));
```

Comment:
Denne støtter kun variabler fra valgfelt, men ikke direkt i en delmal

Reply:
Så rename funksjonen til finnManglendeBrevVariablerFraValgfelt og bare bruke denne funksjonaliteten i første omgang?

Reply:
Har opprettet en ny Favro-oppgave for validering av "stand-alone-brev-variabler" her: https://favro.com/organization/98c34fb974ce445eac854de0/4d617346d79341c7fbd9a40a?card=NAV-23991